addons/web/static/src/views/list/list_renderer.xml

```diff
@@ -176,9 +176,9 @@
                 t-attf-class="o_group_name fs-6 fw-bold {{!group.isFolded ? 'text-900' : 'text-700'}}"
                 t-att-colspan="getGroupNameCellColSpan(group)">
                 <div class="d-flex align-items-center">
-                    <span t-attf-class="o_group_caret fa fa-fw {{group.isFolded ? 'fa-caret-right' : 'fa-caret-down' }} me-1"
+                    <span t-attf-class="o_group_caret fa {{group.isFolded ? 'fa-caret-right' : 'fa-caret-down' }}"
                         t-attf-style="--o-list-group-level: {{getGroupLevel(group)}}"/>
-                    <t t-esc="getGroupDisplayName(group)"/> (<t t-esc="group.count"/>)
+                    <span class="text-truncate ms-2 me-1" t-esc="getGroupDisplayName(group)"/> (<t t-esc="group.count"/>)
```

Comment:
Why do we need to change the XML tag? It does not really respect the stable policy. 😕 

Cannot we fix the issue without changing the XML tag?

Reply:
Yes, the stable policy is being violated. Currently, class is not applied to the `<t>` tag. can we make the invisible?

``<t class="o_data" t-esc="getGroupDisplayName(group)" t-if="1 == 0"/>``